### ISSUE TITLE
branches != files

### ISSUE THREAD

**Issue by pboling** [chetan/simplecov-console#23]
```
COVERAGE:  58.42% -- 267/457 lines in 9 files
BRANCH COVERAGE:   0.00% -- 0/167 branches in 9 branches
```

Should be:

```
COVERAGE:  58.42% -- 267/457 lines in 9 files
BRANCH COVERAGE:   0.00% -- 0/167 branches in 9 files
```

https://github.com/chetan/simplecov-console/blob/master/lib/simplecov-console.rb#L68

**Comment by chetan** [comment]
Nice catch! Surprised this wasn't spotted sooner. I'll get it merged and released tomorrow.

**Comment by pboling** [comment]
:) Fixed by #24
